rbd/rbd_nautilus_test.go

```diff
@@ -77,3 +77,79 @@ func TestClosedImageNautilus(t *testing.T) {
 	_, err = image.GetModifyTimestamp()
 	assert.Error(t, err)
 }
+
+func TestGetParent(t *testing.T) {
+	conn := radosConnect(t)
+	poolName := GetUUID()
+	err := conn.MakePool(poolName)
+	assert.NoError(t, err)
+	ioctx, err := conn.OpenIOContext(poolName)
+	assert.NoError(t, err)
+
+	defer func() {
+		ioctx.Destroy()
+		conn.DeletePool(poolName)
+		conn.Shutdown()
```

Comment:
Could you add assertions here to check errors for the functions that return them? Thanks!